bob/src/cluster/quorum.rs

```diff
@@ -124,6 +154,37 @@ impl Quorum {
         put_at_least(key, data, target_nodes, at_least, PutOptions::new_local()).await
     }
 
+    pub(crate) async fn delete_at_remote_nodes(
+        &self,
+        key: BobKey,
+        without_aliens: bool,
+    ) -> (Vec<NodeOutput<Error>>, usize) {
+        let local_node = self.mapper.local_node_name();
+        let mut target_nodes = vec![];
+        if !without_aliens {
+            target_nodes.extend(self.mapper.nodes().values());
+        } else {
+            target_nodes.extend(self.mapper.get_target_nodes_for_key(key))
+        };
+        target_nodes.retain(|n| n.name() != local_node);
+        debug!(
+            "DELETE[{}] cluster quorum put remote nodes {} total target nodes",
+            key,
+            target_nodes.len(),
+        );
+        let count = target_nodes.len();
+        (
+            delete(
```

Comment:
It would be nice to expand the name to distinguish it from the `self.delete` function